app/src/lib/app-state.ts

```diff
@@ -411,6 +411,15 @@ export interface IBranchesState {
 
   /** The pull request associated with the current branch. */
   readonly currentPullRequest: PullRequest | null
+
+  /**
+   * Is the current branch configured to rebase on pull?
+   *
+   * This is equivalent to this config value in Git:
+   *
+   *  - `git config pull.rebase true` (local or global)
+   */
+  readonly pullWithRebase?: boolean
```

Comment:
It's not clear to me what this property being `undefined` means. Could you add a bit more to the docs to explain?

Reply:
I was able to figure out what it means from reviewing the code below, but I still think a comment of what it means for Desktop would be helpful.

Reply:
See ff462628c for more context